### DISCUSSION TITLE
Change humanise variations

### DISCUSSION THREAD

**Discussion by Insomnia88** [EDCD/EDDI#2195] [EDDI Standalone Q&A] [answered]
I am translating many scripts to my language (german) but there are some things that I am not able to change (they remain english). They are triggered with the "humanise" functions. Is there a way to change them?

**Comment by Tkael** [comment, marked as answer]
Are you aware of our translation project? Please see https://github.com/EDCD/EDDI/blob/develop/docs/Localization_for_translators.md

I've made our lead translator for German aware of this conversation so that they can look into this too.

**Reply by Insomnia88** [reply]
Nope, wasn't aware of it before. I hoped for already translated scripts (even if I wasted hours translating so far) but it seems they are not part of it - just the main gui/programme.
I guess the stuff I am looking for is inside "SpeechService/Properties/Phrases.resx".
I have just registered and will take a look into it :)

**Reply by Tkael** [reply]
There is currently no translation of the default scripts for German. Some translators go as far as to submit a default personality for their language but we recognize that is a big undertaking and quite difficult to maintain at this time.
